controllers/datadoghq/recommender.go

```diff
@@ -14,13 +14,46 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	sigmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
 	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/types/known/structpb"
 
 	autoscaling "github.com/DataDog/agent-payload/v5/autoscaling/kubernetes"
 	"github.com/DataDog/watermarkpodautoscaler/apis/datadoghq/v1alpha1"
 )
 
+var (
```

Comment:
Metrics are currently all aggregated in `metrics.go`.

Reply:
moving